src/mono/wasm/wasm.proj

```diff
@@ -293,7 +293,6 @@
       <_EmccLinkFlags Include="-s EXPORTED_RUNTIME_METHODS=$(_EmccExportedRuntimeMethods)" />
       <_EmccLinkFlags Include="-s EXPORTED_FUNCTIONS=$(_EmccExportedFunctions)" />
       <_EmccLinkFlags Include="--source-map-base http://example.com" />
-      <_EmccLinkFlags Include="-s STRICT_JS=1" />
       <_EmccLinkFlags Include="-s WASM_BIGINT=1" />
```

Comment:
Is this needed/related ?

Reply:
Yes, it is a fix to resolve issue related to regression in the latest emscripten

Reply:
I think that latest emscripten doesn't allow it together with `EXPORT_ES6=1` which is fine because that is always strict. I think @radekdoulik mentioned that last week.